corehq/apps/app_manager/xform.py

```diff
@@ -20,7 +21,7 @@ def parse_xml(string):
     try:
         return ET.fromstring(string, parser=ET.XMLParser(encoding="utf-8", remove_comments=True))
     except ET.ParseError, e:
-        raise XFormException("Error parsing XML" + (": %s" % str(e)))
+        raise XFormException(_("Error parsing XML: %s") % str(e))
```

Comment:
not sure if the translations framework will handle this, but i wonder if all of these should be unicode and not strings